src/comptime.rs

```diff
@@ -5,9 +5,17 @@ use const_soft_float::soft_f64::SoftF64 as Sf64;
 
 /// A `const` matrix type, with dimensions checked at compile time
 /// for all operations.
-#[derive(Clone, PartialEq, PartialOrd, Copy)]
+#[derive(Clone, Copy, PartialEq, PartialOrd)]
 pub struct CMatrix<const R: usize, const C: usize>([[Sf64; C]; R]);
 
+impl<const R: usize, const C: usize> Default for CMatrix<R, C> {
```

Comment:
Currently, arrays don't have const generic Default impl and Default is impl only for arrays of size 32.

https://doc.rust-lang.org/std/primitive.array.html#impl-Default-for-%5BT%3B%2032%5D

Reply:
Thanks, that does remind me I should make an identity initializer at some point

Reply:
I'm curious, so you mean From impl for different arrays by "identity initialiser"?

Reply:
Like [[1, 0], [0, 1]] for `CMatrix::<2,2>`, for example